AnyText/AnyText.Core/Model/AddAssignReferenceRule.cs

```diff
@@ -187,9 +194,9 @@ protected internal override bool CheckIfStillExist(ParseContext context)
                 var parent = (AddAssignReferenceRule<TSemanticElement, TReference>)(RuleApplication.Rule);
                 if (parent.TryResolveReference(contextElement, resolveString, context, out var reference))
                 {
-                    parent.SetResolved(RuleApplication, reference);
+                    /*parent.SetResolved(RuleApplication, reference);
```

Comment:
Da wir jetzt einmal 1. den Wert in die RuleApplication eintragen und ihn dann 2. im ParseContext als Referenz eintragen und die Kombination davon an vielen Stellen, würde sich eine Hilfsmethode dafür wahrscheinlich schon lohnen